src/aspire/commands/orient3d.py

```diff
@@ -29,7 +29,41 @@
     type=int,
     help="Max number of image rows to read from STAR file",
 )
-def orient3d(data_folder, starfile_in, starfile_out, pixel_size, max_rows):
+@click.option(
+    "--n_rad",
+    default=None,
+    type=int,
+    help="Number of points in the radial direction. If None, defaults to half the resolution of the source.",
+)
+@click.option(
+    "--n_theta",
+    default=360,
+    type=int,
+    help="Number of points in the theta direction",
+)
+@click.option(
+    "--max_shift",
+    default=0.15,
+    type=float,
+    help="Maximum range for shifts as a proportion of resolution",
+)
+@click.option(
+    "--shift_step",
+    default=1,
+    type=int,
+    help="Resolution for shift estimation in pixels",
+)
+def orient3d(
+    data_folder,
+    starfile_in,
+    starfile_out,
+    pixel_size,
+    max_rows,
+    n_rad,
+    n_theta,
+    max_shift,
+    shift_step,
+):
     """
     Input images from STAR file and estimate orientational angles
```

Comment:
Agree that this could be made better. It's my understanding that "viewing directions" and what are being called "orientational angles" are slightly different things. The viewing direction corresponds to only the third row of the rotation matrix that describes the particles orientation and also corresponds to the "orientational angles". It's something like `Orientation = viewing direction + in-plane rotation`.

At any rate, maybe something like "Estimate particle orientations of a data source and save orientation matrices"? other suggestions?

Reply:
I like that suggestion, yeah. Did not realize the subtlety, thx